client/my-sites/plans/main.jsx

```diff
@@ -230,6 +234,29 @@ class Plans extends Component {
 		return <ECommerceTrialPlansPage interval={ interval } siteSlug={ selectedSite.slug } />;
 	}
 
+	renderWooExpressMediumPage() {
+		return <p>Woo Express Performance</p>;
+	}
+
+	renderMainContent() {
+		const { selectedSite } = this.props;
+		const currentPlanSlug = selectedSite?.plan?.product_slug;
+
+		switch ( currentPlanSlug ) {
+			case PLAN_ECOMMERCE_TRIAL_MONTHLY:
+				return this.renderEcommerceTrialPage();
+
+			case PLAN_WOOEXPRESS_MEDIUM:
+			case PLAN_WOOEXPRESS_MEDIUM_MONTHLY:
+				if ( isEnabled( 'plans/wooexpress-medium' ) ) {
+					return this.renderWooExpressMediumPage();
+				}
+
+			default:
+				return this.renderPlansMain();
+		}
+	}
```

Comment:
Might it be cleaner to handle this via early returns from multiple conditions? And maybe we can pass the `isWooExpressPlan` and `isECommerceTrial` flags from `render()` as arguments to simplify the logic as well?
```suggestion
	renderMainContent( { isECommerceTrial, isWooExpressPlan } ) {
		if ( isECommerceTrial ) {
			return this.renderEcommerceTrialPage();
		}

		if ( isWooExpressPlan && isEnabled( 'plans/wooexpress-medium' ) ) {
			return this.renderWooExpressMediumPage();
		}

		return this.renderPlansMain();
	}
```